app/Resources/views/base.html.twig

```diff
@@ -44,6 +44,8 @@
         <meta name="twitter:description" content="{{ block('open_graph_description') }}" />
         <meta name="twitter:image:src" content="{{ block('open_graph_image') }}" />
 
+        <link rel="alternate" type="application/rss+xml" title="En Marche!" href="{{ absolute_url(path('articles_feed')) }}" />
```

Comment:
What I meant was that you can probably replace `{{ absolute_url(path('articles_feed')) }}` by {{ url('articles_feed') }} with the same behavior.